### PR TITLE
add new android operation(open app,exit,stop)

### PR DESCRIPTION
**Pull request by kithib**
**Features**

- xx
- yy
    
**Feature Docs**

**Influence**
You can use EXIT to return to the main interface of the mobile phone and turn off the application that is running in the background
You can use stop to return to the task success information
You can use OpenApp without XML to turn on the mobile app without XML

**Result**
截屏2024-04-11 17 57 30

**Other**

### CODE REVIEW

File: metagpt/environment/android/android_ext_env.py [hunk 1]
```diff
@@ -228,3 +233,40 @@ def user_swipe_to(self, start: tuple[int, int], end: tuple[int, int], duration:
...
         return swipe_res
+
+    @mark_as_writeable
+    def user_exit(self):
+        adb_cmd = "adb shell am start -a android.intent.action.MAIN -c android.intent.category.HOME"
+        exit_res = self.execute_adb_with_cmd(adb_cmd)
+        return exit_res
+
+    @mark_as_writeable
+    def user_openApp(self, app_name: str):
```
Comment (geekan) [inline metagpt/environment/android/android_ext_env.py:244]: user_open_app

File: metagpt/environment/android/text_icon_localization.py [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+        )
+        coordinate.append(
+            [max(0, box[0] - 25), max(0, box[1] - 25), min(box[2] + 25, size[0]), min(box[3] + 25, size[1])]
+        )
+
+    return image_data, coordinate
+
+
+if __name__ == "__main__":
+    from modelscope.pipelines import pipeline
```
Comment (geekan) [inline metagpt/environment/android/text_icon_localization.py:391]: add modelscope to requirements

File: metagpt/environment/android/text_icon_localization.py [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+    from modelscope.utils.constant import Tasks
+
+    image_ori = "./screenshot/screenshot.png"
```
Comment (geekan) [inline metagpt/environment/android/text_icon_localization.py:394]: could you add more tests? the image is under android pad i think. but we need some regular android screens

File: metagpt/environment/android/android_ext_env.py (hunk 1 above)
Comment (kithib) [reply metagpt/environment/android/android_ext_env.py]: already edited

File: metagpt/environment/android/text_icon_localization.py (hunk 2 above)
Comment (kithib) [reply metagpt/environment/android/text_icon_localization.py]: already edited

File: metagpt/environment/android/text_icon_localization.py (hunk 3 above)
Comment (kithib) [reply metagpt/environment/android/text_icon_localization.py]: 截屏2024-04-18 11 57 19

**Pull request closed without merging**